manifests/gcp_marketplace/schema.yaml

```diff
@@ -118,69 +118,79 @@ properties:
+      managed storage options as your previous instance.
 
-      If upgrading from managed storage Kubeflow Pipelines 0.5.1, GCS bucket
-      name was previously auto generated in the same project, it should have been
-      named like "<cloudsql instance connection name>-<database prefix or instance name>".
-
-      If false, following options will be ignored and the data will be stored in
-      GCE Persistent Disks.
+      If upgrading from Kubeflow Pipelines 0.5.1 with managed storage, the
+      Cloud Storage bucket was auto-generated in the same project. This bucket
+      should be named like "<cloudsql instance connection name>-<database prefix or instance name>".
     default: false
   managedstorage.gcsBucketName:
     type: string
-    title: Artifact storage GCS bucket (Managed storage only)
+    title: Artifact storage Cloud Storage bucket (Managed storage only)
     description: |-
-      This field must be specified if choose to use managed storage.
-      Provide the Google Cloud Storage (GCS) bucket name for an existing bucket.
-      The bucket will be used for storing the output artifacts.
-      For more details on how to create a new bucket, see https://cloud.google.com/storage/docs/creating-buckets.
+      If you are deploying Kubeflow Pipelines with managed storage, specify the
+      Cloud Storage bucket that you would like Kubeflow Pipelines to store
+      pipeline artifacts in. Learn more about creating a new bucket,
+      see https://cloud.google.com/storage/docs/creating-buckets.
 
-      Warning, if provided bucket doesn't exist, deployer will automatically
-      create a bucket for you in us-central1, but you won't be able to customize
-      any parameters.
+      Warning, if the provided bucket doesn't exist, the deployer will
+      automatically create a bucket for you in us-central1, but you won't be
+      able to customize the bucket's parameters.
   managedstorage.cloudsqlInstanceConnectionName:
     type: string
     title: Cloud SQL instance connection name (Managed storage only)
     description: |-
-      This field must be specified if choose to use managed storage.
-      Provide the instance connection name for an existing Cloud SQL for MySQL instance.
-      The instance connection name can be found on the instance detail page in the Cloud SQL console.
-      The instance connection name uses the format project:zone:instance-name, for example,myproject:us-central1:myinstance.
-      For more details on how to create a new instance, see https://cloud.google.com/sql/docs/mysql/quickstart.
+      If you are deploying Kubeflow Pipelines with managed storage, specify the
+      instance connection name for your MySQL instance on Cloud SQL.
+      The instance connection name can be found on the instance detail page in
+      the Cloud SQL console. The instance connection name uses the format
+      "<project>:<zone>:<instance-name>", for example: myproject:us-central1:myinstance.
```

Comment:
We have a mistake here, it should be "\<project>:\<region>:\<instance-name>", us-central1 is a region. (verified in real data, this is always a region)

Reply:
UPDATE: I edited the PR in-place to change this to region

Reply:
good point, thanks Yuan.